integration/tests/conftest.py

```diff
@@ -548,3 +548,56 @@ def expected_error_checker(accounts, web3_client):
         "common/ExpectedErrorsChecker", "0.8.12", accounts[0], contract_name="A"
     )
     yield contract
+
+
+@pytest.fixture(scope="class")
+def query_account_caller_contract(
```

Comment:
do you use it?

Reply:
It's not necessary anymore for these tests, removed ✅